File: Python/1013.py
line1=input().split(" ")
a, b, c = line1
ab = int( (int(a) + int(b) + abs(int(a)-int(b)))/2 )
print( int((int(ab) + int(c) + abs(int(ab)-int(c)))/2) , "eh o maior" )

File: uriOnlineJudge/cpp/1013.cpp
#include <iostream>
//The greatest
using namespace std;

int ehmaior(int a,int b){
	int maior;
	maior = (a+b+abs(a-b))/2;
	return maior;
}


int main(){
	int a, b, c;
	cin >> a >> b >> c;
	cout << ehmaior(a,ehmaior(b,c)) << " eh o maior" << endl;

	return 0;
}
